app/presenters/reports/repository_presenter.rb

```diff
@@ -1,19 +1,18 @@
 module Reports
   class RepositoryPresenter < SimpleDelegator
-    def self.create(regions, period:, months: Reports::MAX_MONTHS_OF_DATA, current_admin: nil)
+    def self.create(regions, period:, months: Reports::MAX_MONTHS_OF_DATA, use_who_standard: nil)
       start_period = period.advance(months: -(months - 1))
       range = Range.new(start_period, period)
-      repo = Reports::Repository.new(regions, periods: range, current_admin: current_admin)
+      repo = Reports::Repository.new(regions, periods: range, use_who_standard: use_who_standard)
       new(repo)
     end
 
-    def call(region)
-      to_hash(region)
+    def call(region, use_who_standard = nil)
+      to_hash(region, use_who_standard)
     end
 
-    def to_hash(region)
+    def to_hash(region, use_who_standard = nil)
       slug = region.slug
-      use_who_standard = Flipper.enabled?(:diabetes_who_standard_indicator, current_admin)
       {
```

Comment:
I think because this is a `SimpleDelegator`, we should be able to access `use_who_standard` (which would be delegated to the `attr_reader` of `Repository`) directly without needing to pass it in as an argument.

I am not a 100% sure but if that works, we can eliminate needing to take this arg. If that doesn't this is fine.